pom.xml

```diff
@@ -147,12 +147,17 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
```

Comment:
Wild guess: could be that the ice4j version which ends up being used is the one libjitsi depends on. We should update libjitsi's pom.xml and see if that helps.